src/modules/common/components/Dropdown.js

```diff
@@ -0,0 +1,102 @@
+// @flow
+
+import React from 'react'
+import { Wrapper, Button, Menu, MenuItem } from 'react-aria-menubutton'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faAngleDown } from '../../app/constants/icons'
+import type { ThemeType } from '../../theme/constants/theme'
+import styled from 'styled-components'
+
+const StyledWrapper = styled(Wrapper)`
+  position: relative;
+  box-sizing: border-box;
+`
+
+const StyledMenuButton = styled(Button)`
+  display: flex;
+  cursor: pointer;
+  box-sizing: border-box;
+  align-items: center;
+  justify-content: space-between;
+  border-width: 1px;
+  border-style: solid;
+  border-color: ${props => props.theme.colors.textDecorationColor};
+  border-radius: 4px;
+`
+
+const StyledMenuButtonText = styled.div`
+  display: flex;
+  padding: 8px;
+  color: ${props => props.theme.colors.textSecondaryColor};
+`
+
+const StyledMenuButtonIcon = styled.div`
+  padding: 8px 16px;
+`
+
+const StyledMenu = styled(Menu)`
+  position: absolute;
+  top: 100%;
+  z-index: 1;
+  display: flex;
+  width: 100%;
+  box-sizing: border-box;
+  flex-direction: column;
+  margin: 8px 0;
+  border-radius: 4px;
+  background-color: ${props => props.theme.colors.backgroundColor};
```

Comment:
Minor: could be refactored to:

`(props: { theme: ThemeType }) => theme.colors.backgroundColor` 

That way flow types work I think, not sure though. Maybe it also works that way.

Reply:
We don't have this anywhere yet. But I think maybe we have to take a look at flow types for styled components at some time. I'd leave this as it is for now though (only because of consistency of course :stuck_out_tongue: )